### PR TITLE
Add NVIDIA's `dcgm-exporter` to export GPU metrics to Prometheus

### PR DESCRIPTION
**Pull request by linki**
Adds NVIDIA's Data Center GPU Manager (DCGM) Prometheus Exporter as a sidecar to the already configured NVIDIA device plugin and configures Prometheus to scrape its metrics endpoint.

For https://github.bus.zalan.do/zooport/issues/issues/4628

### CODE REVIEW

**Comment by linki** [comment]
👍

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -387,6 +387,10 @@ flannel_memory: "100Mi"
 nvidia_device_plugin_cpu: "10m"
 nvidia_device_plugin_memory: "50Mi"
 
+# nvidia dcgm exporter
+nvidia_dcgm_exporter_cpu: "10m"
+nvidia_dcgm_exporter_memory: "200Mi"
```
Comment (linki) [inline cluster/config-defaults.yaml:392]: Unfortunately, this memory usage is what I observed during my tests.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by linki** [comment]
👍

**Pull request merged**